packages/slice-machine/src/features/builder/StaticFieldsInfoDialog.tsx

```diff
@@ -0,0 +1,77 @@
+import {
+  Box,
+  Dialog,
+  DialogActionButton,
+  DialogActions,
+  DialogContent,
+  DialogHeader,
+  ScrollArea,
+  Text,
+  Video,
+} from "@prismicio/editor-ui";
+import { ReactNode, useState } from "react";
+
+interface StaticFieldsInfoDialogProps {
+  onClose: () => void;
+  trigger?: ReactNode;
+}
+
+export function StaticFieldsInfoDialog(props: StaticFieldsInfoDialogProps) {
+  const { onClose, trigger } = props;
+  const [isOpen, setIsOpen] = useState(false);
+
+  const handleOpenChange = (open: boolean) => {
+    if (!open) {
+      dismissDialog();
+    } else {
+      setIsOpen(open);
+    }
+  };
+
+  function dismissDialog() {
+    setIsOpen(false);
+    onClose();
+  }
+
+  return (
+    <Dialog
+      open={isOpen}
+      onOpenChange={handleOpenChange}
+      trigger={trigger}
+      size="small"
+    >
+      <DialogHeader title="Static zone" />
+      <DialogContent>
+        <ScrollArea>
+          <Box as="article" flexDirection="column" padding={16} gap={4}>
+            <Text sx={{ marginBottom: 4 }} color="purple9" variant="smallBold">
+              About
+            </Text>
+            <Text variant="h3">What is the static zone?</Text>
+            <Text color="grey11">
+              The static zone in Prismic page types contain fields that are
+              always present on the page. Use it for essential information like
+              a page's title, page design settings or metadata. Unlike slices,
+              which can be added or removed, static zone fields remain fixed.
```

Comment:
Since we aren't referring to this as the Static zone anymore, I would change this to match what we call it in the UI, "..., static fields remain fixed."

Reply:
This clears this up: https://github.com/prismicio/slice-machine/pull/1426#discussion_r1715315531